### DISCUSSION TITLE
So slow?

### DISCUSSION THREAD

**Discussion by bar1bar1** [Taxel/PlexTraktSync#1488] [General]
Hi,
I do have a large library but 13 hours seems too long, and not sure if once only or every time.
Can I only synced my watch items?
that can't be more then an hour....I don't understand why the scan have to go through every movie/tv show?
I was hoping for it to access my watched list and sync it....not sure if thats possible at all

**Comment by scoiattolo49** [comment]
!3 hours seems a bit over the top, even for a large library. My library of 1500 movies and about the same TV episodes takes just a few minutes.
I would have a look at the logs to see what it's going on.

**Comment by simonc56** [comment]
First sync is slow because it builds the cache. Next syncs are fast.
Also disable in config.yml what you don't need.
You can give your items/sec rate to check if something's wrong. You can see it in console during the sync.

**Comment by cams87** [comment]
Hi, I hope I'me not hijacking this thread, if I am please telle me I'll delete and create my own.
I also experience some slow syncs... sometimes? I have a 1000+ movies collection and some syncs are done at 45 items/sec and some at 1 or less (40min+). INFO log doesn't give any information that could explain what it's happening. Would DEBUG do?

**Comment by simonc56** [comment]
Show your config.yml and the log output.
If you sync only `watched_status` it can be very fast.  I'm at 200 items/sec. 
items/sec rate is only readable for movies (https://github.com/Taxel/PlexTraktSync/issues/976)

As an example, you can see that syncing `liked_lists` is very slow. So disable it if not needed, or do it only once a month.

**Reply by cams87** [reply]
Hi! I set liked_lists to false but anyway, I don't use this feature so there is nothing to sync. But I found the culprit, each time the sync took ages, my server was actually downloading something in the same time... As my connection download speed is limited, that makes sense. (I'll finally get gigabit on Friday)
Thank you!
